.evergreen/install-dependencies.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+install_extension ()
+{
+   # Workaround to get PECL running on PHP 7.0
+   export PHP_PEAR_PHP_BIN=${PHP_PATH}/bin/php
+   export PHP_PEAR_INSTALL_DIR=${PHP_PATH}/lib/php
+
+   rm -f ${PHP_PATH}/lib/php.ini
+
+   if [ "x${DRIVER_BRANCH}" != "x" ] || [ "x${DRIVER_REPO}" != "x" ]; then
+      CLONE_REPO=${DRIVER_REPO:-https://github.com/mongodb/mongo-php-driver}
+      CHECKOUT_BRANCH=${DRIVER_BRANCH:-master}
+
+      echo "Compiling driver branch ${CHECKOUT_BRANCH} from repository ${CLONE_REPO}"
+
+      mkdir -p /tmp/compile
+      rm -rf /tmp/compile/mongo-php-driver
+      git clone ${CLONE_REPO} /tmp/compile/mongo-php-driver
+      cd /tmp/compile/mongo-php-driver
+
+      git checkout ${CHECKOUT_BRANCH}
+      git submodule update --init
+      phpize
+      ./configure --enable-mongodb-developer-flags
+      make all -j20 > /dev/null
+      make install
+
+      cd ${PROJECT_DIRECTORY}
+   elif [ "x${DRIVER_VERSION}" != "x" ]; then
+      echo "Installing driver version ${DRIVER_VERSION} from PECL"
+      pecl install -f mongodb-${DRIVER_VERSION}
+   else
+      echo "Installing latest driver version from PECL"
+      pecl install -f mongodb
```

Comment:
I know PECL tries to append to `php.ini` after it runs. It looks like that only if `php_ini` is specified in the PEAR/PECL configuration.

If you want to add `pecl config-set php_ini ${PHP_PATH}/lib/php.ini` before invoking `pecl install`, perhaps that would let you do without the custom `php.ini` file in this PR. Although if `${PHP_PATH}/lib/php.ini` doesn't already exist (per my next comment), you'd need to `touch` first.

Maybe that's not worth the trouble since you'd also have to deal with file permissions and/or run `pecl` with `sudo`. Feel free to ignore.

Reply:
Also covered by PHPC-1772, and I'd test this in master and then back port.